### PR TITLE
git-credential-cache: add page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

#3953

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]

File: pages/common/git-credential-cache.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# git credential-cache
+
+> `Git` helper to temporarily store passwords in memory.
```
Comment (kbdharun) [inline pages/common/git-credential-cache.md:3]:
```suggestion
> `git` helper to temporarily store passwords in memory.
```

File: pages/common/git-credential-cache.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# git credential-cache
+
+> `git` helper to temporarily store passwords in memory.
```
Comment (navarroaxel) [inline pages/common/git-credential-cache.md:3]:
```suggestion
> Git helper to temporarily store passwords in memory.
```

File: pages/common/git-credential-cache.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+> More information: <https://git-scm.com/docs/git-credential-cache>.
+
+- Store Git credentials for a specific amount of time:
+
+`git config credential.helper 'cache --timeout={{1..infinity}}'`
```
Comment (navarroaxel) [inline pages/common/git-credential-cache.md:8]: Perhaps this better
```suggestion
`git config credential.helper 'cache --timeout={{time_in_seconds}}'`
```

File: pages/common/git-credential-cache.md (hunk 2 above)
Comment (EmilySeville7cfg) [reply pages/common/git-credential-cache.md:3]: I can't apply them automatically: no such button available for me. :( So, please apply them yourself if u think it makes PR better.

**Pull request closed without merging**